proto/lighthouse-result.proto

```diff
@@ -607,3 +610,36 @@ message StackPack {
   // Map of audit-ids to Stack Pack descriptions.
   map<string, string> descriptions = 4;
 }
+
+// Message containing Entity Classification.
+message EntityClassification {
+  // List of entities.
+  repeated Entity entities = 1;
+  
+  // Name of the first-party entity.
+  string firstParty = 2;
+  
+  // Lookup table that maps URL origins to corresponding index in entities.
+  map<string, double> originLUT = 3;
+
+  // Lookup table that maps entity names to corresponding index in entities.
+  map<string, double> nameLUT = 4;
+}
+
+// Message containing an Entity.
+message Entity {
+  // Name of the entity.
+  string name = 1;
+  
+  // An optional homepage URL of the entity.
+  optional string homepage = 2;
+
+  // An optional category name for the entity.
+  optional string category = 3;
+
+  // An optional flag indicating if the entity is the first party.
+  optional google.protobuf.BoolValue isFirstParty = 4;
```

Comment:
why not bool?

Reply:
I think here it won't make much of a difference, as both 
  1. defaults as `false` are acceptable when omitted during serialization as a scalar `bool` value (Both `isFirstParty` and `isUnrecognized` are designed to be omitted when not set, to reduce transport size), and
  2. for the same reason, likely nobody needs to find out if `isFirstParty` was specifically set `false` or omitted. 

In general (and to the extent of my protobuf skills), wrappers avail better handling of scalars + default values, especially in cases scalars are serialized to be removed when they hit default value. This one could be `optional bool` I think?